test/development/app-dir/react-performance-track/react-performance-track.test.ts

```diff
@@ -17,9 +18,10 @@ describe('react-performance-track', () => {
 
   it('should show fetch', async () => {
     const browser = await next.browser('/fetch')
+    await browser.elementByCss('[data-react-server-requests-done]')
 
-    const track = await browser.eval('window.reactServerRequests')
-    // FIXME: Should show await fetch and await response.json()
+    const track = await browser.eval('window.reactServerRequests.getSnapshot()')
+    // FIXME: Should show await fetch
```

Comment:
There's actually a source of flakiness in here. If we haven't streamed in the body when we call `response.json()`, it'll show up as a Server Request. If it is already downloaded, `response.json()` is just a microtask and will be dropped.